### PR TITLE
Upgrade bundled SIMDe to 0.7.6

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

File: src/sfizz/Interpolators.hpp [hunk 1]
```diff
@@ -64,7 +64,7 @@ class Interpolator<kInterpolatorHermite3, float>
         simde__m128 x = simde_mm_sub_ps(simde_mm_setr_ps(-1, 0, 1, 2), simde_mm_set1_ps(coeff));
         simde__m128 h = hermite3x4(x);
         simde__m128 y = simde_mm_mul_ps(h, simde_mm_loadu_ps(values - 1));
-        return simde_vaddvq_f32(y);
+        return simde_vaddvq_f32(simde_vld1q_f32(simde__m128_to_private(y).f32));
```
Comment (mr-c) [inline src/sfizz/Interpolators.hpp]: Is this codepath always or often on an SSE capable x86/amd64 system? If so , then I can add a shortcut as well.

**Comment by redtide** [comment]
Thank you a lot for this. Sorry but I'm not a SIMDe expert, so I'm just checking from time to time and allow the builds to run

**Comment by mr-c** [comment]
> Thank you a lot for this. Sorry but I'm not a SIMDe expert, so I'm just checking from time to time and allow the builds to run

No worries! I'll experiment on my fork a bit longer before I push here again

**Comment by mr-c** [comment]
@redtide This should pass now, sorry for the noise! https://github.com/mr-c/sfizz/actions/runs/4918541525/jobs/8785010604

**Comment by redtide** [comment]
Thank you for the noise!

**Pull request merged**